packages/tag/Tag.tsx

```diff
@@ -12,7 +10,16 @@ export class Tag extends Component<TagProps> {
   static get is() { return 'bl-tag' }
 
   private handleClick() {
-    emit(this,'tagClose')
+    emit( this, 'tagClose', {
```

Comment:
Let's not use strings, we all hate them right ? :) 

What I do is this design pattern:
```ts
class MyMama extends Component<Props>{
  static get is(){ return 'yo-mama' }
  static get events(){ return { TAG_CLOSE: 'tagClose' } )
  ....
  private handleClick() {
     emit(this,MamaComponent.events.TAG_CLOSE, payload);
  }
}
```